tests/e2e/helpers/decompressors/brotli-database-populator.ts

```diff
@@ -0,0 +1,134 @@
+import * as proto from 'protobufjs';
+import { pack as msgpackrPack } from 'msgpackr';
+import * as brotli from 'brotli-unicode';
+import * as fflate from 'fflate';
+import * as fs from 'fs';
+import { BaseDatabasePopulator } from './base-decompressors-populator';
+
+const COMPRESSED_PREFIX = 'Comp';
+const BROTLI_PREFIX = 'BROTLI';
+
+export class BrotliDatabasePopulator extends BaseDatabasePopulator {
+
+    /**
+     * Create keys with all types of Bolti compression
+     */
+        protected async createCompressedKeys(): Promise<void> {
+        await this.createBrotliUnicodeKeys();
+        await this.createBrotliASCIIKeys();
+        await this.createBrotliVectorKeys();
+        await this.createBrotliJSONKeys();
+        await this.createBrotliPHPUnserializedJSONKeys();
+        await this.createBrotliJavaSerializedObjectKeys();
+        await this.createBrotliMsgpackKeys();
+        await this.createBrotliProtobufKeys();
+        await this.createBrotliPickleKeys();
+    }
+
+    private async createBrotliUnicodeKeys() {
```

Comment:
what about comments with description for all your methods?

Reply:
I believe that for private methods, especially those with self-explanatory names, comments may not always add significant value. it just clutters up the code

Reply:
okay